### ISSUE TITLE
Improve accessibility for students

### ISSUE THREAD

**Issue by Kaabero** [UniversityOfHelsinkiCS/gptwrapper#665]
- [ ] Where focus should be when the application starts? The current focus order is not very user-friendly. Maybe an option in the left sidebar to move focus directly to the chat box?
- [x] Focus after starting a new chat: Move focus to the chat input
- [x] #688
- [x] #698
- [x] #685
- [ ] Token usage menu: Auto focus after opening the menu. Inform the user that menu is open. Question-mark icon next to token usage should receive keyboard focus and the modal it opens should be accessed via keyboard. Precentages should be announced.
- [x] Model selector menu: Auto focus after opening the menu. Inform the user that menu is open. 
- [ ] Use of alert components: Do not use the alert component for visual UI purposes. Alerts are announced immediately by screen readers, so they should only be used when the information requires an accessibility announcement.
- [ ] Announce notifications and error messages immediately
- [x] #696
- [ ] Add file upload/remove success announcements
- [ ] Add description of how to remove an attachment
- [x] Better Toska link description
- [ ] PromptModal: Inform the user that the modal is open and describe the modal. Auto focus selected prompt. Inform the user when prompt or course is selected for preview.
- [ ] PromptList: Tell if the promptlist is opened or closed. Announce that the filter menu is open and auto focus the first element.
- [ ] Add description/label to course/prompt buttons in prompt modal  and move focus to course/prompt preview after selecting course/prompt to preview
- [x] copy prompt modal: focus handling
- [x] #697
- [x] Chat: Announce if the server didn't respond in time etc.
- [ ] User should be able to change uni prompt language for preview 
- [ ] focus the open sidebar button after closing the sidebar

Investigate:
- [ ] Course view
- [ ] Uni prompts
- [ ] Search results sidebar
- [ ] Main menu options: settings, give feedback, info
- [ ] Saving the conversation 
- [ ] Mui Button/Link use
- [ ] heading hierarchy (the use of variant and component)
- [ ] IconButtons
